.gitlab-ci.yml

```diff
@@ -30,6 +31,7 @@ build:image:
   script:
     - bin/build
     - mkdir -p docker/images
+    - ARNOLD_IMAGE="arnold:$(tr -d '\n' < VERSION)"
```

Comment:
Why not using YAML anchors to avoid repetition?

Reply:
No time for this for the moment but yes we must use anchors.
I'm open an issue for sync circleCI and gitlab-ci files.